app/src/main/java/net/skyscanner/backpack/demo/meta/StoriesRepository.kt

```diff
@@ -28,13 +28,19 @@ import org.jetbrains.annotations.TestOnly
 
 interface StoriesRepository {
 
-  fun allComponents(): List<Component>
+  fun uiComponents(): List<Component>
+
+  fun tokenComponents(): List<Component>
 
   fun screenshotStories(): List<Story>
 
-  fun storiesOf(component: Component, compose: Boolean): List<Story>
+  fun storiesOf(component: String, compose: Boolean): List<Story>
+
+  fun storyOf(component: String, story: String, compose: Boolean): Story
+
+  fun isComposeOnly(component: String): Boolean
 
-  fun isComposeSupportedFor(component: Component): Boolean
+  fun isViewOnly(component: String): Boolean
```

Comment:
Since almost of the the components support compose now, it'll be better to indicate view-only or compose-only components